### PR TITLE
lerna versioning

### LINKED ISSUE #602: bleeding edge validator shows version 0.0.0
When using the bids-validator in the master repo and the /bin directory, it yields 0.0.0 as its version.

`bids-validator/bin/bids-validator --version`

Shouldn't this be 1.0.1.dev (whatever the current version is) or similar?

### PR DESCRIPTION
**Pull request by david-nishi**
Introduces [Lerna](https://github.com/lerna/lerna) to maintain the version numbers for the bids-validator and bids-validator-web packages separately and within their package.json files (should resolve #602).

Publish a new version with `yarn lerna publish` and follow instructions to set major/minor/patch updates.

### CODE REVIEW

**Review by nellh — CHANGES_REQUESTED** [review]
The documentation needs to be updated to include info on Lerna (at least a link to it) and document how to do a release.

File: bids-validator/README.md [hunk 1]
```diff
@@ -297,3 +297,7 @@ is useful to run tests while making changes. A coverage report is available with
 `npm run coverage`.
 
 To run the linter which checks code conventions run `npm run lint`.
+
+#### Publishing
```
Comment (sappelhoff) [inline bids-validator/README.md:301]: is publishing used synonymously with "making a release" here?

Comment (nellh) [reply bids-validator/README.md:301]: Using `lerna publish` will create a git commit with updated version information and create a version number tag for it, push the tag to GitHub, then publish to NPM and PyPI. The GitHub release is manual following that.

Comment (sappelhoff) [reply bids-validator/README.md:301]: Okay :+1: thanks for clarifying! Perhaps we could add exactly that information.

**Review by nellh — APPROVED** [review]

**Review by rwblair — APPROVED** [review]

**Pull request merged**